packages/form-js-carbon-styles/src/carbon-styles.js

```diff
@@ -1084,7 +1085,7 @@ const DROPDOWN_STYLES = css`
 
       & .fjs-dropdownlist-item:hover,
       & .fjs-dropdownlist-item.focused {
-        background-color: var(--cds-layer-hover, var(--cds-layer-hover-01));
+        background-color: var(--cds-layer-hover);
```

Comment:
I also took the opportunity to remove the unnecessary fallbacks

Reply:
I think while experimenting with the styles, I was able to create setups where the `cds-layer-hover` without a specific number wasn't specified, so that's why these fallbacks landed in initially. However, I think I pretty much misconfigured something, so it's very unlikely that this will happen.